www/include/options/accessLists/groupsACL/DB-Func.php

```diff
@@ -68,7 +68,7 @@ function testGroupExistence($name = null)
     }
     $query = "SELECT acl_group_id, acl_group_name "
         . "FROM acl_groups "
-        . "WHERE acl_group_name = '" . htmlentities($name, ENT_QUOTES, "UTF-8")."' ";
+        . "WHERE acl_group_name = '" . htmlentities($name, ENT_QUOTES, "UTF-8")."' "; //Seriously ???? CentreonDB::escape($name)
```

Comment:
120 chars max per line  AND please remove useless comments
the code isn't a chat box -> contact us using our mails, our slack community or using github comments feature... but not like this

Reply:
done